PyInstaller/loader/rthooks/pyi_rth_kivy.py

```diff
@@ -0,0 +1,10 @@
+from os.path import join
+from os import environ
+import sys
+
+root = join(sys._MEIPASS, 'kivy_install')
+sys.path += [join(root, '_libs')]
```

Comment:
Hmm, this means that you are putting module into the file-system instead into the PYZ archive. Does kivi not support frozen applications? If it would, we'd more elegantly put these modules into the PYZ, too.